airlock/file_browser_api.py

```diff
@@ -43,10 +43,16 @@ class PathNotFound(Exception):
     # should this node be expanded in the tree?
     expanded: bool = False
 
+    # Is this a supporting file?
+    supporting_file: bool = False
+
     # what to display for this node when rendering the tree. Defaults to name,
     # but this allow it to be overridden.
     display_text: str = None
 
+    # Additional html classes that can be added to this node
+    extra_html_classes: list[str] = field(default_factory=list)
```

Comment:
Is this used anywhere now?

Reply:
It is not. I'm pretty sure I removed this, maybe it crept back in in a rebase